index.html

```diff
@@ -440,15 +445,6 @@ <h3>DataIntegrityProof</h3>
 The `cryptosuite` property of the proof MUST be `bbs-2023`.
           </p>
           <p>
-The `created` property of the proof MUST be an [[XMLSCHEMA11-2]]
-formatted date string.
-          </p>
-          <p>
-The `proofPurpose` property of the proof MUST be a string, and MUST
-match the verification relationship expressed by the verification method
-`controller`.
-          </p>
-          <p>
 The value of the `proofValue` property of the proof MUST be an BBS signature or
 BBS proof produced according to [[CFRG-BBS-SIGNATURE]] then serialized and encoded
 according to procedures in section <a href="#algorithms"></a>.
```

Comment:
I think this little tweak is more clear:

```suggestion
The value of the `proofValue` property of the proof MUST be a BBS signature or
BBS proof produced according to [[CFRG-BBS-SIGNATURE]] that is serialized and encoded
according to procedures in section <a href="#algorithms"></a>.
```